lib/controllers/messages.ts

```diff
@@ -61,6 +61,8 @@ import {
   MessageModel,
 } from "../models/message";
 
+const ApplicationInsightsClient = ApplicationInsights.defaultClient;
```

Comment:
`defaultClient` [is null unless you call setup()](https://github.com/Microsoft/ApplicationInsights-node.js/blob/develop/applicationinsights.ts#L38) (that creates a new client)